lua/oil/view.lua

```diff
@@ -630,7 +634,7 @@ local function render_buffer(bufnr, opts)
       jump_idx = #line_table
       M.set_last_cursor(bufname, nil)
     end
-    ::continue::
+   end)()
```

Comment:
Since this loop doesn't have that many lines, I'd prefer to just nest it under `if M.should_display(entry[FIELD_NAME], bufnr) then` instead of wrapping it in a function